freesound/urls.py

```diff
@@ -104,6 +104,9 @@
 
     url(r'^contact/', support.views.contact, name="contact"),
     url(r'^search/$', search.views.search, name='sounds-search'),
+    url(r'^clustering_facet/$', search.views.clustering_facet, name='clustering-facet'),
+    url(r'^clustering_graph_visualisation/$', search.views.cluster_visualisation, name='clustering-graph-visualisation'),
```

Comment:
what's the purpose of this page? Who needs to view it? Does it make sense to make it admin-only? Should it have a different URL if that's the case?